app/models/division.rb

```diff
@@ -50,8 +50,7 @@ def self.root_internal_name
 
   def self.root
     # create on demand if not present for convenience of blank db's and test cases
-    # @@root ||=
```

Comment:
Re seeds.rb.  I'll strip it down to just a generic root division without an assigned ID and nothing else.  The ID's were only important in the context of the migrated system, so that need is now handled by the static_data.rb code.  I'll let you add countries, languages or currencies if/when you see fit.